### PR TITLE
Bazel intellij Issue Template

### PR DESCRIPTION
**Pull request by sgowroji**
# Checklist

- [ Yes] I have filed an issue about this change and discussed potential changes with the maintainers.
- [ Yes] I have received the approval from the maintainers to make this change.
- [ Yes] This is not a stylistic, refactoring, or cleanup change.

Please note that the maintainers will not be reviewing this change until all checkboxes are ticked. See 
the [Contributions](https://github.com/bazelbuild/intellij#contributions) section in the README for more 
details.

# Description of this change

This PR provides the templates for type:bug and type:feature in bazelbuild/Intellij Repository.

### CODE REVIEW

**Review by jastice — APPROVED** [review]
Thank you!

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Bug Report
+description: File a bug report
+body:
+  - type: markdown
+    attributes:
+      value: >
+        **Attention:** if this is a _question_ about how to build / test / query / deploy using
+        Bazel intellij, or a _discussion starter_, send it to bazel-discuss@googlegroups.com or discuss on
```
Comment (mai93) [inline .github/ISSUE_TEMPLATE/bug_report.yml:8]: I think we should direct the users to this group instead: [intellij-bazel-plugin@googlegroups.com](mailto:intellij-bazel-plugin@googlegroups.com)

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      label: >
+        What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if
+        possible.
+  - type: input
+    id: os
+    attributes:
+      label: >
+        Which Intellij IDE are you using? Please provide the specific version.
+  - type: input
+    id: version
```
Comment (mai93) [inline .github/ISSUE_TEMPLATE/bug_report.yml:27]: Maybe the id should be updated to something else, e.g. tools?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      label: >
+        Description of the bug:
+  - type: textarea
+    id: repro
+    attributes:
+      label: >
+        What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if
+        possible.
+  - type: input
+    id: os
```
Comment (mai93) [inline .github/ISSUE_TEMPLATE/bug_report.yml:22]: Maybe this id should be replaced with something else like product?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        What Bazel plugin version are you using?
+  - type: textarea
+    id: relevant-info
+    attributes:
+      label: Have you found anything relevant by searching the web?
+      description: >
+        Places to look:
+
+         * GitHub issues: https://github.com/bazelbuild/intellij/issues
+         * Email threads on https://groups.google.com/forum/#!forum/bazel-discuss
```
Comment (mai93) [inline .github/ISSUE_TEMPLATE/bug_report.yml:44]: I think we should also update this to https://groups.google.com/forum/#!forum/intellij-bazel-plugin

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
+name: Feature Request
+description: Suggest a new feature
+body:
+  - type: markdown
+    attributes:
+      value: >
+        **Attention:** if this is a _question_ about how to build / test / query / deploy using
+        Bazel intellij, or a _discussion starter_, send it to bazel-discuss@googlegroups.com or discuss on
```
Comment (mai93) [inline .github/ISSUE_TEMPLATE/feature_request.yml:8]: Also here we should direct the users to this group: [intellij-bazel-plugin@googlegroups.com](mailto:intellij-bazel-plugin@googlegroups.com)

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+        What operating system, Intellij IDE and programming languages are you using? Please provide specific versions.
+  - type: textarea
+    id: relevant-info
+    attributes:
+      label: Have you found anything relevant by searching the web?
+      description: >
+        Places to look:
+
+         * GitHub issues: https://github.com/bazelbuild/intellij/issues
+         * Email threads on https://groups.google.com/forum/#!forum/bazel-discuss
```
Comment (mai93) [inline .github/ISSUE_TEMPLATE/feature_request.yml:33]: We should also update this to https://groups.google.com/forum/#!forum/intellij-bazel-plugin

**Review by mai93 — APPROVED** [review]

**Pull request merged**